pkg/server/plugin/nodeattestor/tpmdevid/selectors.go

```diff
@@ -0,0 +1,63 @@
+package tpmdevid
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"fmt"
+
+	spc "github.com/spiffe/spire/proto/spire/common"
+)
+
+var attributeTypeNames = map[string]string{
+	"2.5.4.3":  "cn",           // Common name
+	"2.5.4.5":  "serialnumber", // Serial number
+	"2.5.4.6":  "c",            // Country
+	"2.5.4.8":  "st",           // State
+	"2.5.4.10": "o",            // Organization
+	"2.5.4.11": "ou",           // Organizational unit
```

Comment:
I'd prefer to not deal with codepoints like this. The pkix name struct given by golang x509 certificate has all of this stuff already and we can create selectors from them explicitly?

I also wonder if we really need all of these. Selector naming and count is a general concern, and it would be good to have some clarity on the need before adding a bunch of them. The x509pop attestor emits only fingerprints and cn, and we have not had any request for more than that... it has been there for some time now.

To be clear, I'm all for having more selectors if it's useful to have them. What I'm trying to avoid is supporting selectors that users may be less likely to consume.

Reply:
I talked to some folks which are planning to use this plugin internally. They seem to be ok with using the subject common name and fingerprint. So I think that we should be fine generating the same set of selectors that are generated in x509pop (at least for this initial version). 
I'll update this.